docs/modules/operation/nav.adoc

```diff
@@ -140,8 +140,12 @@
 ** xref:flows/sentinel/sentinel.adoc[]
 ** xref:flows/classification-engine.adoc[]
 ** xref:flows/aggregation.adoc[]
+** xref:flows/nephron.adoc[]
```

Comment:
@Bonrob2 Should this link be part of `foundation-2022`?

Reply:
No, ultimately, once we have the /latest feature merged, we will link to the nephron documentation which is in a separate repo, and will appear as a separate component like Helm once it's integrated.